client/src/templates/Challenges/redux/completion-epic.js

```diff
@@ -175,15 +175,9 @@ export default function completionEpic(action$, state$) {
 }
 
 async function findPathToNavigateTo(nextChallengePath, superBlock) {
-  let canClaimCert = false;
-  let pathToNavigateTo;
-
-  if (nextChallengePath.includes(superBlock) && !canClaimCert) {
-    pathToNavigateTo = nextChallengePath;
-  } else if (canClaimCert) {
-    pathToNavigateTo = `/learn/${superBlock}/#claim-cert-block`;
+  if (nextChallengePath.includes(superBlock)) {
+    return nextChallengePath;
   } else {
-    pathToNavigateTo = `/learn/${superBlock}/#${superBlock}-projects`;
+    return `/learn/${superBlock}/#${superBlock}-projects`;
```

Comment:
Technically, this does not work for `/2022/responsive-web-design`, because there is no `2020/responsive-web-design-projects` block/card

Reply:
However, a Camper does not know, because the redirect is still to the correct page.